driver/razermouse_driver.c

```diff
@@ -76,6 +76,10 @@ static int razer_get_report(struct usb_device *usb_dev, struct razer_report *req
         return razer_get_usb_response(usb_dev, 0x00, request_report, 0x00, response_report, RAZER_VIPER_MOUSE_RECEIVER_WAIT_MIN_US, RAZER_VIPER_MOUSE_RECEIVER_WAIT_MAX_US);
         break;
 
+    case USB_DEVICE_ID_RAZER_BASILISK_V3:
+        return razer_get_usb_response(usb_dev, 0x03, request_report, 0x03, response_report, RAZER_MOUSE_WAIT_MIN_US, RAZER_MOUSE_WAIT_MAX_US);
```

Comment:
Unfortunate as this is, it looks to be required. `wIndex == 0x03` seems to be a necessary condition for the device to respond to reports correctly - without this [I just got `-ETIMEDOUT` back from the device](https://github.com/openrazer/openrazer/issues/1610#issuecomment-991357746).

Reply:
Great that you figured this out. But maybe adjust the code to be similar to the razerkbd driver: https://github.com/openrazer/openrazer/blob/master/driver/razerkbd_driver.c#L256
Like report_index/response_index variable there but it's probably enough to introduce one variable (called `index`?) because we haven't seen a device that needs different values there and I'm not sure if we ever will.

Reply:
Have applied this. Change looks pretty safe but I don't have other devices to regression test with, so may be a good idea to double check with another device as this is a pretty central function to touch.